packages/atoms/src/Radio.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import {
+    NativeStyleObject,
+    NativeStyleUtils,
+    prepareNativeStyle,
+    useNativeStyles,
+} from '@trezor/styles';
+import { Pressable, View } from 'react-native';
+
+type RadioValue = string | number | undefined;
+
+interface RadioProps {
+    value: string;
+    isChecked?: boolean;
+    onPress?: (value: RadioValue) => void;
+    isDisabled?: boolean;
+    style?: NativeStyleObject;
+}
+
+type RadioStyleProps = {
+    isChecked?: boolean;
+    isDisabled?: boolean;
+};
+
+const getRadioBorderColor = (
+    utils: NativeStyleUtils,
+    { isChecked, isDisabled }: RadioStyleProps,
+) => {
+    if (isChecked && isDisabled) {
+        return {
+            borderColor: utils.colors.gray400,
+        };
+    }
+    if (isChecked) {
+        return {
+            borderColor: utils.colors.green,
+        };
+    }
+    return {
+        borderColor: utils.colors.gray400,
+    };
+};
+
+const radioStyle = prepareNativeStyle<RadioStyleProps>((utils, { isChecked, isDisabled }) => ({
+    height: 24,
+    width: 24,
+    backgroundColor: isDisabled ? utils.colors.gray400 : utils.colors.white,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: utils.borders.radii.round,
+    borderWidth: utils.borders.widths.md,
```

Comment:
I think the border is **2px** in figma, so it should be `widths.lg` :)

Reply:
You have a good eye! ;) It is 1.5 (md) for not checked button and 2 (lg) for checked. Thank you - fixed.